gc/verbose/schema.xsd

```diff
@@ -120,6 +120,14 @@ xmlns:vgc="http://www.ibm.com/j9/verbosegc">
 		<attribute name="micro-fragmented" type="integer" use="optional" />
 		<attribute name="macro-fragmented" type="integer" use="optional" />
 	</attributeGroup>
+
+	<attributeGroup name="region">
+		<attribute name="regionSize" type="integer" use="required" />
+		<attribute name="regionCount" type="integer" use="required" />
+		<attribute name="arrayletLeafSize" type="integer" use="required" />
+		<attribute name="arrayletDoubleMapping" type="integer" use="required" />
+		<attribute name="arrayletDoubleMappingStatus" type="integer" use="optional" />
+	</attributeGroup>
```

Comment:
Does not look right. We'll have to follow the existing rules, sorry for confusing you earlier.

```
	<complexType name="attribute">
		<attribute name="name" type="string" />
		<attribute name="value" type="string" />
	</complexType>
```

which means we wont be able to report both 'requested' and 'status' and one line, but in 2 lines. One with attribute name arrayletDoubleMappingRequested and values true/false and andother onewith name arrayletDoubleMapping and with values enabled/disabled

Reply:
There is nothing to add to schema.xsd then

Reply:
or there is, the max should be bumped here:

```
	<complexType name="region">
		<sequence maxOccurs="1" minOccurs="1">
			<element ref="vgc:attribute" maxOccurs="3" minOccurs="0" />
		</sequence>
	</complexType>	
```